camlp4/default.toml

```diff
@@ -0,0 +1,3 @@
+# Use this file to templatize your application's native configuration files.
```

Comment:
If the plan doesn't run a service, we don't need to commit the default.toml. Just leave all of these out.